data/data.json

```diff
@@ -6332,7 +6332,7 @@
    "coredns": "rancher/coredns-coredns:1.7.0",
```

Comment:
Can you confirm that we are only using coredns:1.7.0 and not anything higher ? See: https://github.com/k3s-io/k3s/issues/2700#issuecomment-747743431

Reply:
```
$ curl -s https://releases.rancher.com/kontainer-driver-metadata/release-v2.5/data.json | jq -r '.K8sVersionRKESystemImages[] | select(.coredns != null) | .coredns' | sort -u
coredns/coredns:1.2.2
coredns/coredns:1.2.6
coredns/coredns:1.3.1
rancher/coredns-coredns:1.2.2
rancher/coredns-coredns:1.2.6
rancher/coredns-coredns:1.3.1
rancher/coredns-coredns:1.6.2
rancher/coredns-coredns:1.6.5
rancher/coredns-coredns:1.6.9
rancher/coredns-coredns:1.7.0
$ curl -s https://releases.rancher.com/kontainer-driver-metadata/release-v2.4/data.json | jq -r '.K8sVersionRKESystemImages[] | select(.coredns != null) | .coredns' | sort -u
coredns/coredns:1.2.2
coredns/coredns:1.2.6
coredns/coredns:1.3.1
rancher/coredns-coredns:1.2.2
rancher/coredns-coredns:1.2.6
rancher/coredns-coredns:1.3.1
rancher/coredns-coredns:1.6.2
rancher/coredns-coredns:1.6.5
rancher/coredns-coredns:1.6.9
```